ui/lib/core/addon/components/autocomplete-input.hbs

```diff
@@ -11,7 +11,7 @@
       class="input"
       {{on "input" this.onInput}}
     />
-    <BasicDropdown @registerAPI={{fn (mut this.dropdownAPI)}} @renderInPlace={{true}} as |D|>
```

Comment:
is this removed because mut is being deprecated?

Reply:
No `mut` is still a good helper but it was throwing the _twice in the same computation_ error but didn't when I moved it to an action.